config/logicsoft/ZHC5010.xml

```diff
@@ -0,0 +1,216 @@
+                </Value>
+
+                <Value type="list" genre="config" instance="1" index="23" label="Control of association groups for device 1 (button #1)" min="0" max="1" value="1" size="1" >
+                        <Help>Control of association groups for device 1 (button #1)</Help>
+                        <Item label="When commands are received by device 1, nodes in the association groups will not be switched, button activations will still send switch values." value="0" />
+                        <Item label="When commands are received by device 1, nodes in the association groups will be switched." value="1" />
+                </Value>
+
+                <Value type="list" genre="config" instance="1" index="24" label="Control of association groups for device 2 (button #2)" min="0" max="1" value="1" size="1" >
+                        <Help>Control of association groups for device 2 (button #2)</Help>
+                        <Item label="When commands are received by device 2, nodes in the association groups will not be switched, button activations will still send switch values." value="0" />
+                        <Item label="When commands are received by device 2, nodes in the association groups will be switched." value="1" />
+                </Value>
+
+                <Value type="list" genre="config" instance="1" index="25" label="Control of association groups for device 3 (button #3)" min="0" max="1" value="1" size="1" >
+                        <Help>Control of association groups for device 3 (button #3)</Help>
+                        <Item label="When commands are received by device 3, nodes in the association groups will not be switched, button activations will still send switch values." value="0" />
+                        <Item label="When commands are received by device 3, nodes in the association groups will be switched." value="1" />
+                </Value>
+
+                <Value type="list" genre="config" instance="1" index="26" label="Control of association groups for device 4 (button #4)" min="0" max="1" value="1" size="1" >
+                        <Help>Control of association groups for device 4 (button #4)</Help>
+                        <Item label="When commands are received by device 4, nodes in the association groups will not be switched, button activations will still send switch values." value="0" />
+                        <Item label="When commands are received by device 4, nodes in the association groups will be switched." value="1" />
+                </Value>
+
+        </CommandClass>
+<commandClass id="0085" version="2"/>
```

Comment:
Thanks for taking a look at this!
Sorry for not replying, I haven't had a lot of time lately.
you mean line 171 -> 187 should just be removed?
I will do that, and test it,
though when trying to edit, I get a 404 on github at the moment, hopefully that will be resolved shortly.